scripts/setup-env.sh

```diff
@@ -127,3 +127,6 @@ echo ""
 alias python=python3
 
 export HDMI2USB_ENV=1
+
+# Set prompt
+PS1="(HDMI2USB)$PS1"
```

Comment:
@mithro I had `basename $PWD`, but it made the prompt too big.

Will the user have the env variables for driving make set when running setup-env.sh?